src/main/java/hudson/tasks/EnhancedMessage.java

```diff
@@ -0,0 +1,41 @@
+package hudson.tasks;
+
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.MimeMessage;
+import java.io.InputStream;
+
+public class EnhancedMessage extends MimeMessage
```

Comment:
Restrict the class via `@Restricted(NoExternalUse.class)`. Also, please use the `jenkins.plugins.mailer.tasks` package for new code

Reply:
Added both changes.